panos/policies.py

```diff
@@ -44,6 +44,7 @@ class Rulebase(VersionedPanObject):
         "policies.PolicyBasedForwarding",
         "policies.SecurityRule",
         "policies.DecryptionRule",
+        "policies.AuthenticationRule",
```

Comment:
Please add this line to PreRulebase and PostRulebase also.

Reply:
Thanks for your feedback. I don't understand why ?

Reply:
`policies.Rulebase` is for NGFW rules.  But Panorama has `policies.PreRulebase` and `policies.PostRulebase`.  So if it's not there in those two, then the rule is not useable / accessible for Panorama.

Reply:
Yes but `policies.PreRulebase` and `policies.PostRulebase` inherits from `policies.Rulebase` (just adding an _xpaths profile. 
It works actually pretty well with Panorama !